### DISCUSSION TITLE
Nextcloud AIO set up for a while now; Wondering how to properly set it up behind NGINX Proxy Manager

### DISCUSSION THREAD

**Discussion by d-ognyanoff** [nextcloud/all-in-one#3613] [Questions] [answered]
Sorry for the noob questions, but I seem to be going in circles in the documentation.

I went trough this https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#nginx-proxy-manager
and also https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#3-limit-the-access-to-the-apache-container

Do I have to remove the mastercontainer and run:

```
sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 8080:8080 \
--env APACHE_PORT=11000 \
--env APACHE_IP_BINDING=0.0.0.0 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
```
When I do this I can't seem to be able to access my cloud.domainname.com. 

Whenever I follow the instructions for the setup in Nginx Proxy Manager (to request a new SSL) I get "Internal Error".

What is the proper approach with a setup running instance and and a new Nginx Proxy Manager?

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?
